sep/026: Add task to fix sepolia fp upgrade (#421)
ethereum-optimism/superchain-ops: NESTED-VALIDATION.md

  **Meaning:** This is `owners[1] -> 0xca11bde05977b3631167028862be2a173976ca11`, so the key can be
    derived from `cast index address 0x0000000000000000000000000000000000000001 2`.

## State Changes

The following state changes related to the nested Safe execution must be seen, either for the
Security Council, or the Foundation Safe, depending on which role the simulation was run for:

### `GnosisSafeProxy` - `approvedHashes` mapping update

- **Key:** _Needs to be computed._ <br/>
  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000`<br/>
  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>

#### Key Computation

The GnosisSafe `approvedHashes` mapping is updated to indicate approval of this transaction by the chosen Safe Role. The correctness of this slot can be verified as follows:
- Since this is a nested mapping, we need to use `cast index` twice to confirm that this is the correct slot. The inputs needed are:
    - The location (`8`) of the `approvedHashes` mapping in the [GnosisSafe storage layout](https://github.com/safe-global/safe-contracts/blob/v1.4.0/contracts/libraries/SafeStorage.sol#L23)

>     - The address of the Safe Role, stored at the env var `$SAFE_ROLE` in the following cast script command.
>     - The safe hash to approve, stored at the env var `$SAFE_HASH` in the following cast script command.
>       It's the value after "Nested hash:" in the simulation output logs.
> - Then using `cast index`, we can compute the key with
>     ```shell
>       $ cast index bytes32 $SAFE_HASH $(cast index address $SAFE_ROLE 8)
>     ```
>     The output of this command must match the key of the state change.

Reviewer: Is this suggestion accurate? Safe Role was a bit unclear to me
```suggestion
    - The address of the Safe signer, stored at the env var `$SAFE_SIGNER` in the following cast script command.
    - The safe hash to approve, stored at the env var `$SAFE_HASH` in the following cast script command.
      It's the value after "Nested hash:" in the simulation output logs.
- Then using `cast index`, we can compute the key with
    ```shell
      $ cast index bytes32 $SAFE_HASH $(cast index address $SAFE_SIGNER 8)
    ```
    The output of this command must match the key of the state change.
```

Reviewer: This is not a blocker, feel free to ignore this change, because I see in https://github.com/ethereum-optimism/superchain-ops/pull/425 we use the same terminology and I don't want to create unnecessary rework

Author: I've introduced a definition of `Safe Role` further up in the doc. But I'm happy to change the definition to `Safe Signer`.
